alpha/lib/model/flavorAsset.php

```diff
@@ -405,4 +405,21 @@ public function getTypeFolderName()
 		return 'flavors';
 	}
 
+	/**
+	 * @return int|null
+	 * value is in milliseconds
+	 */
+	public function getSegmentDuration()
+	{
+		$tags = explode(',', $this->getTags());
+		foreach ($tags as $tag)
+		{
+			if (strpos($tag, 'segment_duration:') !== false)
+			{
+				return (int)explode(':', $tag)[1];
```

Comment:
Better be safe and verify there is indeed a value after the colon ':'
Else it would throw a warning of array key missing